Contributors.md

```diff
@@ -0,0 +1,3 @@
+Contributor:
+
+Ong Yean Lin
```

Comment:
There is no need of creating a contributors page. I'll mention each contributor in the readme itself.